extension.js

```diff
@@ -204,6 +217,23 @@ class Player {
 	}
 }
 
+class Status {
```

Comment:
Rename to getStatus for consistency

Reply:
yeah, I called it status for consistency with "Player". Ideally, I would have liked to combine the Metadata and PlaybackStatus checks but couldn't seem to get it to work without essentially duplicating the whole Player class...